browser/lib/wakatime-plugin.js

```diff
@@ -0,0 +1,51 @@
+import config from 'browser/main/lib/ConfigManager'
+const exec = require('child_process').exec
+const path = require('path')
+let lastHeartbeat = 0
+
+function sendWakatimeHeartBeat(
+  storagePath,
+  noteKey,
+  storageName,
+  isWrite,
+  hasFileChanges,
+  isFileChange
+) {
```

Comment:
The last 3 variable is all booleans. I suggest we change to object type for more clarification. For example, if we change this function to:
```js
function sendWakatimeHeartBeat(storagePath, noteKey, storageName, {isWrite, hasFileChanges, isFileChange}) {
  ...
}
```
We can keep the code inside the same but when we call this function, we will call it like this:

```js
sendWakatimeHeartBeat(
  storagePath,
  noteKey,
  storageName,
  { isWrite: true, hasFileChanges: false, isFileChange: true}
)
```